MainRepository/Program.cs

```diff
@@ -1,7 +1,44 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Spell Bee Program
+var words = File.ReadAllLines ("c:/etc/words.txt");
+char[] seed = { 'U', 'X', 'L', 'T', 'A', 'E', 'N' };
+List<string> valid = new ();
+foreach (var word in words)
+   if (IsValid (word)) valid.Add (word);
+for (int i = 0; i < valid.Count - 1; i++) {
+   for (int j = i + 1; j < valid.Count; j++) {
+      if (InOrder (valid[i], valid[j])) {
+         (valid[j], valid[i]) = (valid[i], valid[j]);
+      }

```

Comment:
Single statement don't require { }
```suggestion
      if (InOrder (valid[i], valid[j]))
         (valid[j], valid[i]) = (valid[i], valid[j]);

```